deployment.rst

```diff
@@ -207,6 +207,36 @@ Don't forget that deploying your application also involves updating any dependen
 (typically via Composer), migrating your database, clearing your cache and
 other potential things like pushing assets to a CDN (see `Common Post-Deployment Tasks`_).
 
+Troubleshooting
+---------------
+
+Deployments not Using the ``composer.json`` File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Symfony applications provide a ``kernel.project_dir`` parameter and a related
+:method:`Symfony\\Component\\HttpKernel\\Kernel\\Kernel::getProjectDir>` method
+so you can perform file operations relative to your project's root directory.
```

Comment:
[...] method. You can use this method to perform operations with file paths relative to your project's root directory. [...]